src/javascript/app/base/header.js

```diff
@@ -365,11 +367,49 @@ const Header = (() => {
             }
         });
 
+        // Language Popup.
+        const current_language = Language.get();
+        const available_languages = Object.entries(Language.getAll()).filter(language => !(/ACH/.test(language[0])));
+
+        const $language_select = getElementById('language-select');
```

Comment:
The `$` character suggests these are jQuery elements, but they're not. You can prefix the variable with instead `el_`, e.g.` el_language_select`.